unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -198,6 +205,10 @@ export class DgraphDataSource extends RESTDataSource {
       return { ...entity, type, children };
     };
 
+    entity.properties = entity.properties?.map((property) => ({
```

Comment:
Oh, sorry I lied, I think there is a small problem with the mSupply query - `getEntities` also needs this mapping I think (as well as `getEntity`)!

Reply:
Thanks! d636da195